### PR TITLE
WIP: Split pages into controllers, introduce navigation

### PR DESCRIPTION
**Pull request by kjdchapman**
This is a pretty big PR, but lots of it is just moving markup around.

This PR doesn't yet cover getting the new navigation on all the pages (I'm thinking that should be a different PR). 

I've got the bare bones of the navigation on just the landing page for now, could pull it out if it makes this change easier to understand. Links are working, but styling's rubbish.

image

### CODE REVIEW

File: app/views/ips/_activation_notice.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="govuk-inset-text">
   <h4 class="govuk-heading-s">New IPs are not immediately available</h4>
   <p class="govuk-body">
-    You can configure your access points now, but new IPs will not be activated until the next day.
+    You can configure your authenticator(s) now, but new IPs will not be activated until the next day.
```
Comment (emileswarts) [inline app/views/ips/_activation_notice.html.erb]: Don't really want to get too much into terminology now but how did we decide on authenticators?
Wouldn't infrastructure be more descriptive to network administrators?

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <%= link_to "Team Members", team_members_path, class: "govuk-link" %>
+    <%= link_to "Settings", ips_path, class: "govuk-link" %>
+  </nav>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Get GovWifi in your building</h2>
+
+    <h3 class="govuk-heading-m"> 1. Configure your authenticator(s) </h3>
```
Comment (emileswarts) [inline app/views/home/index.html.erb]: Is the whitespace within the html tags on purpose?
Seeing quite a lot of this.

Comment (emileswarts) [reply app/views/home/index.html.erb]: Oh I see this was copied over from the previous markup.  Bizarre, need to fix this but can do in a later PR

Comment (kjdchapman) [reply app/views/home/index.html.erb]: Yeah, not on purpose. Want a linter or something to catch this stuff.

File: spec/features/view_ip_addresses_spec.rb [hunk 3]
```diff
@@ -40,43 +40,11 @@
...
-          expect(page).to have_content('RADIUS')
-          expect(page).to have_content('London')
-          expect(page).to have_content('Dublin')
-        end
-
-        it 'displays the correct IPs' do
-          expect(page).to have_content(radius_ip_1)
-          expect(page).to have_content(radius_ip_2)
-          expect(page).to have_content(radius_ip_3)
-          expect(page).to have_content(radius_ip_4)
```
Comment (emileswarts) [inline spec/features/view_ip_addresses_spec.rb]: We don't display this anymore?

File: app/views/ips/_activation_notice.html.erb (hunk 1 above)
Comment (emileswarts) [reply app/views/ips/_activation_notice.html.erb]: Sure yeah

**Pull request merged**